doc/languages-frameworks/agda.section.md

```diff
@@ -4,7 +4,7 @@
 
 Agda can be installed from `agda`:
 ```
-$ nix-env -iA agda
+$ nix-env -f '<nixpkgs>' -iA agda
```

Comment:
This depends on how one has set up their nixpkgs channels (if at all). So maybe add a sentence "..., assuming you have a `nixpkgs-channel` called `nixpkgs`.

Reply:
Yet another possibility would be `nix-env -iA nixos.agda`.

This is the command suggested by https://search.nixos.org/packages?channel=unstable&show=agda&from=0&size=30&sort=relevance&query=agda, and also by the shell:

```
$ agda
The program ‘agda’ is currently not installed. You can install it by typing:
  nix-env -iA nixos.agda
```

Reply:
And yet another side of the coin is that imperative installations are sort of discouraged in the community. Often people will simply update their `configuration.nix`. Or they have a development `shell.nix` in their agda project. So maybe installation for these use cases should be mentioned as well?

Reply:
To be honest I never use `nix-env` for anything, is there a better way we can word this part of the documentation

Reply:
Maybe simply say "Agda is available as the package `agda`" and link to e.g. https://search.nixos.org/packages?channel=unstable&show=agda&from=0&size=30&sort=relevance&query=agda?

Reply:
I think that makes sense, If someone has got this far through the nixpkgs manual, they probably don't want a tutorial on how to use `nix-env`

Reply:
> And yet another side of the coin is that imperative installations are sort of discouraged in the community. Often people will simply update their `configuration.nix`. Or they have a development `shell.nix` in their agda project. So maybe installation for these use cases should be mentioned as well?

I'm working on a few code snippets to be added to the Agda section of the manual.

Reply:
> This depends on how one has set up their nixpkgs channels (if at all). So maybe add a sentence "..., assuming you have a `nixpkgs-channel` called `nixpkgs`.

IMO one can assume they know, if they changed the default channel.

Reply:
I would also like to update the [Agda manual](https://agda.readthedocs.io/en/v2.6.1.1/getting-started/installation.html#nixos) and to make the nix setup more accessible to non-nix-believers to make nix the default Agda built tool some day.

How about updating the installation command in Agda manual, adding a reference to the Nix manual section and replacing the command in the Nix manual with a reference to Nix search (like @turion suggested)?

Reply:
> How about updating the installation command in Agda manual, adding a reference to the Nix manual section and replacing the command in the Nix manual with a reference to Nix search (like @turion suggested)?

Sounds good :)